lib/engine/game/g_1822_ca/game.rb

```diff
@@ -591,14 +591,14 @@ def event_open_detroit_duluth!
         def destination_city(hex, entity)
           return hex.tile.cities[0] unless (exits = entity.destination_exits)
 
-          cities = exits.each_with_object([]) do |exit, cities_|
+          dest_cities = exits.each_with_object([]) do |exit, cities|
             hex.paths[exit].each do |path|
               next unless (city = path.city)
 
-              cities_ << city unless cities.include?(city)
```

Comment:
`cities.include?(city)` throws ` "undefined method include for Nil"` because it should have been `cities_.include?(city)`